example/friend-bot.ts

```diff
@@ -62,13 +62,13 @@ bot
  */
 .on('friend', async (contact, request) => {
   let logMsg
-  const m = new Message()
-  m.set('to', 'filehelper')
+  const fileTransfer = Contact.load('filehelper')
 
   try {
     logMsg = 'received `friend` event from ' + contact.get('name')
-    m.set('content', logMsg)
-    bot.send(m)
+    if (fileTransfer) {
```

Comment:
`if` is not necessary here.

Because the `Contact.load()` will always return the contact, or it will throw exception.